__tests__/app.test.js

```diff
@@ -117,3 +115,60 @@ describe("/api/articles", () => {
     });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  describe("GET", () => {
+    test("status 200 returns an array of comments with correct properties, with the most recent comments first - article no. 3", () => {
+      return request(app)
+        .get("/api/articles/3/comments")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.comments.length).toBe(2);
+          expect(body.comments).toBeSortedBy("created_at", {
+            descending: true,
+          });
+          body.comments.forEach((comment) => {
+            expect(comment).toEqual({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              votes: expect.any(Number),
+              article_id: expect.any(Number),
+            });
+          });
+        });
+    });
+    test("status 200 returns an array of comments with correct properties, with the most recent comments first - article no. 1", () => {
```

Comment:
This is testing the same behaviour as the previous test just with a different article, so is not needed